maldoca/base/file.cc

```diff
@@ -223,6 +229,16 @@ absl::Status GetContents(const std::string& path, std::string* contents) {
     }
     absl::StrAppend(contents, absl::string_view(buf, ret));
   }
+  std::string content_tmp = *contents;
+  // base64 decode file if required. However, ignore textproto files.
+  if (path.find(kFileNameBase64EncodingIndicator) != std::string::npos &&
```

Comment:
you're supposed to use absl::StrContains rather than find()

Reply:
done